install/index.markdown

```diff
@@ -150,29 +150,37 @@ title: MoveIt 1 Binary Install
           ROS Melodic - Experimental
         </h3>
         <p>
-         Open an elevated ROS Command Window as described in the installation instructions. Use Chocolatey to Download the ROS on Windows with MoveIt packages. Chocolatey should be installed after following the ROS on Windows instructions.
+         Open an elevated ROS Command Window as described in the installation instructions. Use Chocolatey to Download the ROS on Windows with MoveIt packages:
         </p>
         <div class="bash-command">
           <code>
             mkdir c:\opt\chocolatey<br/>
             set ChocolateyInstall=c:\opt\chocolatey<br/>
             choco source add -n=ros-win -s="https://roswin.azurewebsites.net/api/v2" --priority=1<br/>
+          </code>
+        </div>
+        <p>
+         Chocolatey should be installed after following the ROS on Windows instructions:
```

Comment:
`choco upgrade ros-melodic-moveit`
doesn't install chocolatey, but it installs (or upgrades) moveit. chocolatey needs to be installed according to the instructions [here](http://wiki.ros.org/Installation/Windows#noetic.2FInstallation.2FWindows.Install_Windows_Package_Manager).

Reply:
What I had meant by that sentence in the initial PR was that the linked ROS on Windows instructions include a step to install Chocolatey (and thus it should be installed after following the instructions).

Perhaps it would be clearer in past tense: Chocolatey should have been installed while following the ROS on Windows instructions.

Reply:
Updated per both your feedback.